addons/account/tests/test_payment_term.py

```diff
@@ -397,3 +397,25 @@ def assert_payment_term_values(expected_values_list):
                     'discount_amount_currency': 362.30,
                 },
             ])
+
+    @users('__system__')
```

Comment:
```suggestion
```
We don't really need the decorator.
It is generally intended for portal flows.
We can simply distinguish the active company vs. record (move) company here IMO.